proposals/simd/SIMD.md

```diff
@@ -790,7 +791,8 @@ def S.load_splat(memarg):
 * `i64x2.load32x2_s(memarg) -> v128`: load two 32-bit integers and sign extend each one to a 64-bit lane
 * `i64x2.load32x2_u(memarg) -> v128`: load two 32-bit integers and zero extend each one to a 64-bit lane
 
-Fetch consecutive integers up to 32-bit wide and produce a vector with lanes up to 64 bits.
+Fetch consecutive integers up to 32-bit wide and produce a vector with lanes up
+to 64 bits. The natural alignment is 8.
```

Comment:
I think we should specify "bytes" here, the memarg in spec uses log2, so it can be a bit confusing.